src/commands/CmdNews.cpp

```diff
@@ -500,6 +530,28 @@ void CmdNews::version2_6_0 (std::vector<NewsItem>& items) {
   items.push_back(holidata_2022);
 }
 
+void NewsItem::version3_0_0 (std::vector<NewsItem>& items) {
+  Version version("3.0.0");
+  NewsItem sync {
+    version,
+    /*major=*/true,
+    /*title=*/"New data model and sync backend",
+    /*bg_title=*/"",
+    /*background=*/"",
+    /*punchline=*/
+    "The sync functionality for Taskwarrior has been rewritten entirely, and no longer\n"
+    "supports taskserver/taskd. Instead, the recommended solution is a cloud-storage backend,\n"
+    "although taskchampion-sync-server is also available. As part of this change, the\n"
```

Comment:
As discussed elsewhere, I think you could reword this in a way that doesn't cause as much backlash by only saying *why* the cloud storage backend is better. Words like "robust" and "immature" are more to the point than "recommended". Recommendations are based on values and I would recommend people choose the sync solution that is best aligned with their own values.